tests/ProductTest.php

```diff
@@ -239,21 +241,23 @@ public function processInitialDataProvider()
             // Product initial data provided, item should have an id and appropriate texts fields (description, meta description, etc.)
             array(
                 '1',
+                2,
```

Comment:
`$productNameId` is integer here, but string in other cases, what is the correct data type?

Reply:
According to annotation of `\Findologic\Plentymarkets\Product::setProductNameFieldId` it should be integer, so please change the other cases of the data provider.